es74x-cloud/src/main/scala/tech/beshu/ror/proxy/es/clients/HighLevelClientBasedClusterAdminClient.scala

```diff
@@ -0,0 +1,279 @@
+/*
+ *     Beshu Limited all rights reserved
+ */
+package tech.beshu.ror.proxy.es.clients
+
+import monix.execution.Scheduler
+import org.elasticsearch.action._
+import org.elasticsearch.action.admin.cluster.allocation._
+import org.elasticsearch.action.admin.cluster.health._
+import org.elasticsearch.action.admin.cluster.node.hotthreads._
+import org.elasticsearch.action.admin.cluster.node.info._
+import org.elasticsearch.action.admin.cluster.node.reload.NodesReloadSecureSettingsRequestBuilder
+import org.elasticsearch.action.admin.cluster.node.stats._
+import org.elasticsearch.action.admin.cluster.node.tasks.cancel._
+import org.elasticsearch.action.admin.cluster.node.tasks.get._
+import org.elasticsearch.action.admin.cluster.node.tasks.list._
+import org.elasticsearch.action.admin.cluster.node.usage._
+import org.elasticsearch.action.admin.cluster.repositories.cleanup._
+import org.elasticsearch.action.admin.cluster.repositories.delete._
+import org.elasticsearch.action.admin.cluster.repositories.get._
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Usage of get on optional type.](https://app.codacy.com/app/scarduzio/elasticsearch-readonlyrest-plugin/pullRequest?prid=4591272)